### PR TITLE
Update fr.m3u

### PR DESCRIPTION
**Pull request by boomski**

### CODE REVIEW

**Review by freearhey — CHANGES_REQUESTED** [review]

File: channels/fr.m3u [hunk 1]
```diff
@@ -41,6 +41,8 @@ http://51.254.215.79/appflux/gongmax/GONGMAX/gongmax_1280/chunks.m3u8
 http://51.254.215.79/appflux/gongmax/GONGMAX/gongmax_1280/chunks.m3u8?fluxustv.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/ZEA3az8.png" group-title="",Grand Lille TV
 https://live.creacast.com/grandlilletv/smil:grandlilletv.smil/chunklist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/NMjuEXC.png" group-title="",Gulli
+https://gulli-hls-live.akamaized.net/gulli.isml/gulli-audio102_fre=64000-video=900000.m3u8
```
[the remark this answers is not in the document]
Comment (boomski) [reply channels/fr.m3u]: For me it plays in VLC without problems

[the remark this answers is not in the document]
Comment (boomski) [reply channels/fr.m3u]: Can you play the stream on the website itself ?
https://replay.gulli.fr/Direct

[the remark this answers is not in the document]
Comment (freearhey) [reply channels/fr.m3u]: no, they just show me this for 10 seconds:

Screenshot 2019-10-08 at 19 40 22

I don’t understand French, but I think it says that they won’t show me anything else.

[the remark this answers is not in the document]
Comment (boomski) [reply channels/fr.m3u]: Probably stream is geoblocked ,its not available in your territory is the message.
I watch it from Belgium here it works , strange for a channel from France that i can watch it over here.

I Know we don't add geo streams but could we make a seperate playlist, for example geoblock.m3u and add all these links in there then user from some countries can enjoy those streams there , We could place before the channel name the country code.

For example in this case :
#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/NMjuEXC.png" group-title="",FR | Gulli
https://gulli-hls-live.akamaized.net/gulli.isml/gulli-audio102_fre=64000-video=900000.m3u8

And if they stop working i'm sure  some users will make notice off it , we can make some changes. What do you think?

[the remark this answers is not in the document]
Comment (freearhey) [reply channels/fr.m3u]: I understand you, but the problem is a little more complicated. The fact is that in most cases broken links are found not by users, but by scripts. 

A couple of times a month I run a script on my computer that automatically goes through all 8000 links and checks if broadcasts are available. And for this script there is no difference the channel is geo-blocked or just does not work. The same thing with the VLC player, usually it does not even produce any errors.

At the same time, people from completely different places can use these playlists, and adding broadcasts limited to one/two countries to them seems a little unjustified to me.

That’s why I so much resist adding such links.

**Review by freearhey — APPROVED** [review]

File: channels/fr.m3u (hunk 1 above)
[the remark this answers is not in the document]
Comment (cheznewa) [reply channels/fr.m3u]: > I Know we don't add geo streams but could we make a seperate playlist, for example geoblock.m3u and ...

i'm for this geoblock.m3u, for channel are geoblocked for any country only, please create it.

[the remark this answers is not in the document]
Comment (freearhey) [reply channels/fr.m3u]: @cheznewa You can find out more about why we don't add geoblocked streams to the repository [here](https://github.com/iptv-org/iptv/issues/379#issuecomment-540117333) and [here](https://github.com/iptv-org/iptv/issues/480)

**Pull request merged**